opening_hours.js

```diff
@@ -70,6 +70,23 @@
                 "Kerstmis - Noël"                                          : [ 12, 25 ]
             }
         }, /* }}} */
+        'pl': { /* {{{ */
+            'PH': { // https://pl.wikipedia.org/wiki/Dni_wolne_od_pracy_w_Polsce
+                "Nowy Rok"                                      : [ 1, 1 ],
+                "Święto Trzech Króli"                           : [ 1, 6 ],
```

Comment:
vs. `'Trzech Króli'                   : [ 1, 10 ],` in #154. Wikipedia is saying "6 stycznia" so I guess you are right on this?